raiutils/raiutils/models/model_utils.py

```diff
@@ -60,6 +62,19 @@ def is_forecaster(model):
             model.forecast is not None)
 
 
+def is_object_detection_model(model):
+    """Check if the model is an object detection model.
+
+    :return: True if the model is a an object detection model,
+    False otherwise.
+    :rtype: bool
+    """
+    return (model is not None and
+            hasattr(model, SKLearn.PREDICT) and
+            hasattr(model, SKLearn.PREDICT_PROBA) and
+            model.predict_proba is not None)
```

Comment:
why does this function determine whether a model is object_detection ? doesn't it just ensure that predict and predict_proba are defined or am i missing something ?